node/node_test.go

```diff
@@ -514,6 +515,7 @@ func TestNodeNewNodeCustomReactors(t *testing.T) {
 		proxy.DefaultClientCreator(config.ProxyApp, config.ABCI, config.DBDir()),
 		DefaultGenesisDocProviderFunc(config),
 		DefaultDBProvider,
+		ipfs.Mock(),
```

Comment:
What is surprising to me: if this is a mock node, where do all these badgerv3 related go-routines etc come from in the failing CI test (search for `TestNodeNewNodeCustomReactors` in here: https://gist.github.com/liamsi/927b70a1c80b5859be70b9b03e687250)

Reply:
Didn't we merge Badger work from Mark?